src/data/HomePage.yaml

```
@@ -53,51 +53,171 @@ tours:
 
 
 features:
-  - title: High-level Design
-    content: |
-      ## Yes
-      Some discussion about the feature and maybe a code block
-    code:
-
-  - title: Nested Hyperparameters
-    content: |
-      ## Some discussion of the feature
-      Bla Bla
+  - title: Choosing Models
+    content: >-
+      A Model Registry stores detailed metadata for [over 200 models](/machines) and
+      documentation can be searched without loading model code. 
     code: |
-      # 1. Load the Model
-      Tree = @iload DecisionTreeClassifier
-
-      # 2. Create an instance of the model
-      tree = Tree()
-
-      # 3. Wrap the model and data in a machine
-      mach = machine(tree, X, y)
-
-      # 4. Fit the machine
-      fit!(mach)
-
-      # 5. Make predictions
-      yhat = predict(mach, X)
-
-  - title: Neural Networks
-    content: Ding ding.
-    code: 
-
-  - title: Language Models
-    content: Cool
-    code:
+      # match models to data:
```

Do you think there could be value in something like:
```suggestion
      # Identify models suitable for your data
```
Because reading `match models to data` as someone learning MLJ for the first time would feel a little ambiguous to me (in terms of what `matching` exactly implies).